infrastructure/build/fab/fcm_configuration.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
```

Comment:
•	The name {{{FcmExtract}}} makes me think it will use FCM to perform an extract. A name less open to confusion might be {{{FcmConfiguration}}}. After all FCM only supports one kind of configuration file, the fact that someone decided to put only extract details in it is their choice.